### ISSUE TITLE
[Design] Create final wireframes for short-term referral location solution

### ISSUE THREAD

**Issue by cgednalske** [department-of-veterans-affairs/va.gov-team#91938]
## Background

[Discovery/design ticket with short-term referral location direction](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90909)

Figma: [short-term referral location approach](https://www.figma.com/design/DsRXEFiYLCFnY5nBkp9Dc4/CC-Referral-%7C-Appointments-FE?node-id=3980-9088&t=IIsRgCIUATq69ZEC-1)

## Tasks

- [x] Review background and direction decided from [low-fi design/discovery ticket and UAE UX discussions](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90909)
- [x] Create final wireframes of the [short-term approach](https://www.figma.com/design/DsRXEFiYLCFnY5nBkp9Dc4/CC-Referral-%7C-Appointments-FE?node-id=3980-9088&t=IIsRgCIUATq69ZEC-1) we'll be moving forward with
- [ ] Work with Laura to confirm content for Pending name change (Requests and referrals) and new h2 headings
- [ ] Get appropriate approval for Pending name change (ask Kristen who needs to be involved)
- [ ] Update all CC Figma files with new design

## AC

- [ ] Figma files updated with new design
- [ ] Content finalized and approved
- [ ] Reviewed with UX and Kristen

[Design Link](https://www.figma.com/design/DsRXEFiYLCFnY5nBkp9Dc4/CC-Referral-%7C-Appointments-FE?node-id=3980-9089&t=KHAVW8Qvv8zNGwIn-1)

**Comment by cgednalske** [comment]
@benbrasso-agile6 @Cieramaddox here's the next ticket for the high fidelity versions of the short-term referral location solution.

**Comment by benbrasso-agile6** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Appointments-CC-Direct-Scheduling-660abc13699bfa00195d685a/issues/department-of-veterans-affairs/va.gov-team/91938?planning-poker) with Zenhub @cgednalske @Cieramaddox

**Comment by benbrasso-agile6** [comment]
@cgednalske Sorry, keep bugging you across tickets. I was trying to locate if we had a long-term referral location solution ticket? If so, I'd like begin placing that in the correct sprint column.

**Comment by cgednalske** [comment]
I just talked to Ciera about the presentation and we also chatted about the short-term/long-term solutions so she knows what to do with her ticket. Difficult to explain everything here, but essentially the long-term things we need to think about now best fit within her short-term ticket. There are some constraints we didn't know about before that cause long-term design to be closer to the short term design than we thought, so some of it will go hand-in-hand. 

So to summarize, Ciera will finish the short term and think about the related long-term things. No need for another long-term ticket now, but there could be one that comes out of her short term ticket.

**Comment by cgednalske** [comment]
Ciera's going to make it an item to discuss next week at one of our UX syncs.

**Comment by benbrasso-agile6** [comment]
Thanks, sounds good

**Comment by benbrasso-agile6** [comment]
@cgednalske FYI, here's that ticket

**Comment by Cieramaddox** [comment]
@benbrasso-agile6 These concepts are done here. Do we want to close it and create a new ticket for getting content changes approved?

**Comment by benbrasso-agile6** [comment]
Nice! You want them up into the MVP scope page? 

Can create content ticket in the morning.

**Comment by benbrasso-agile6** [comment]
@cgednalske assume you chatted with Ciera about this yesterday and were good with the solution too?

**Comment by benbrasso-agile6** [comment]
Content ticket - https://app.zenhub.com/workspaces/appointments-fe-ux-5fff340c2d80a4000fb6f69c/issues/gh/department-of-veterans-affairs/va.gov-team/93024
